### PR TITLE
Update URLEncode

### PR DESCRIPTION
**Pull request by Xcreen**
urldecode.json:
- Added Related to URLEncode

urlencode.json:  
- Write URL uppercase (like in URLDecode)  
- Added Member-Function  
- Added Related to URLDecode  
- Added Charsets

### CODE REVIEW

**Comment by shaedrich** [comment]
Supplements #546 and #558.

**Comment by pfreitag** [comment]
Hi @Xcreen thanks for this pull request.

We have decided based on results of a poll to start functions with lowercase, so you can change it back to `urlEncode` 

Otherwise looks good!

**Comment by pfreitag** [comment]
FYI related to the casing @KamasamaK has written a JSON schema for the docs which is currently running on the travis builds, but it is not failing builds yet because we have a few things to fix still.

**Comment by Xcreen** [comment]
@pfreitag Okay i changed it to start with lowercase.  
What i noticed is that Syntax-Highlighting for the Examples disappear when changing it to lowercase...

File: data/en/urlencode.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
 	"name": "urlEncode",
 	"type": "function",
 	"syntax": "urlEncode(string [, charset] [, force])",
+	"member": "string.urlEncode([charset] [, force])",
 	"returns": "string",
-	"related": ["encodeForURL"],
-	"description": "Returns an url-encoded string with respect to charset",
+	"related": ["encodeForURL", "URLDecode"],
```
Comment (shaedrich) [inline data/en/urlencode.json]: There you have missed one uppercase starting "`URLDecode`".

**Comment by pfreitag** [comment]
thanks!

**Pull request merged**